codereadyworkspaces/tl500-devfile-v2.yaml

```diff
@@ -96,7 +96,7 @@ components:
           name: ide-8084
           protocol: http
           targetPort: 8084
-      image: 'quay.io/rht-labs/stack-tl500:3.0.16'
+      image: 'quay.io/eformat/stack-tl500:3.0.17'
```

Comment:
can we keep this as `rht-labs` since it'll be pushed after the PR merged? 🚀

Reply:
lolz !! good spotting ;)